doc/release/release.howto.txt

```diff
@@ -398,58 +481,96 @@ login and copy the contents of your
+Maven artifacts are here:
+    https://repository.apache.org/content/repositories/orgapachecommons-1556/
 
-Tag URL:
-  https://git-wip-us.apache.org/repos/asf?p=commons-numbers.git;a=commit;h=4581a4520315657a4219b37c81f5db80e4a4e43c
+These are the artifacts and their hashes:
 
-Commit ID the tag points at:
-  4581a4520315657a4219b37c81f5db80e4a4e43c
+commons-numbers-1.0-bin.tar.gz=5e61ee22980a7534fb87793bb16eac371d7aea703552f2c8f14f049ae9646f97db8ae7e0e021c6ef4b4f1c78b529e33995fd4893030280437baf3081d429957a
+commons-numbers-1.0-bin.zip=4805949e5fab3c016d35be692b79a587b60e35a5809cc5be0817bce224a9462a5891400085040803183c54f2ff71ebdb385bf5119c5148b801333168aec14444
+commons-numbers-1.0-src.tar.gz=08656e681624e2bf2434714f6060e6b02de673089cc3711a01dfc3f0640791feb689206f68dc323a7bd14cb7760770567edc36bcdbea38b7eb31407521de6316
+commons-numbers-1.0-src.zip=5e80cfe64207c5ce990e2bee3e6a7ab7d22260f9e8f28b936197276c02bcba0e771b23f3f06bdcb4984adee69b748c4ebd3cddea40530b9cf8def64b6d4c91f7
+
+(no need for .asc hashes!)
+I have tested this with:
+
+'mvn clean install site' using:
+
+***
+<mvn -version>
+***
+
+Details of changes since 1.0-beta1 are in the release notes:
+    https://dist.apache.org/repos/dist/dev/commons/numbers/1.0-RC1/RELEASE-NOTES.txt
 
 Site:
-  http://home.apache.org/~erans/commons-numbers-1.0-RC6-site
+    https://home.apache.org/~mattjuntunen/commons-numbers-1.0-RC1-site/
+    (note some *relative* links are broken and the 1.0 directories are not yet created - these will be OK once the site is deployed.)
+
+CLIRR Report:
+    N/A
 
-Distribution files (committed at revision 17284):
-  https://dist.apache.org/repos/dist/dev/commons/numbers/
+JApiCmp Report:
+    N/A
 
-Distribution files hashes (SHA1):
-  c5e70a523160ed848194eeba0efa2b14d23c7a61 commons-numbers-1.0-bin.tar.gz
-  66d7701afc90aafa4c4b6e033b8df6cd84365161 commons-numbers-1.0-bin.zip
-  ef56543c8882a0e4771de83ecf8b3be190f122cf commons-numbers-1.0-src.tar.gz
-  3dc56b0793d8bbd703a4efa68547eb58f6fb5b35 commons-numbers-1.0-src.zip
+RAT Report:
+    https://home.apache.org/~mattjuntunen/commons-numbers-1.0-RC1-site/rat-report.html
 
-KEYS file to check signatures:
+KEYS:
   https://www.apache.org/dist/commons/KEYS
 
-Maven artifacts:
-  https://repository.apache.org/content/repositories/orgapachecommons-1229/
+Please review the release candidate and vote.
+This vote will close no sooner that 72 hours from now.
 
-Please select one of the following options:
- [ ] +1 Release it.
- [ ] +0 Go ahead; I don't care.
- [ ] -0 There are a few minor glitches: ...
- [ ] -1 No, do not release it because ...
+  [ ] +1 Release these artifacts
+  [ ] +0 OK, but...
+  [ ] -0 OK, but really should fix...
+  [ ] -1 I oppose this release because...
 
-This vote will be open for at least 72 hours, i.e. until 2016-12-10T00:00:00Z
-(this is UTC time).
+Thank you,
+
+Matt Juntunen,
+Release Manager (using key 7DD53AEFEDF1C3D392B51EBE346F4FCECFB70B1A)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded AWS Secret Access Key ID. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)